src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -173,7 +173,7 @@ const DiscoverySearch = ({
               <MemoIcArrowBackWhite
                 width={20}
                 height={12}
-                fill={COLORS.white2}
+                fill={COLORS.white}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `MemoIcArrowBackWhite` component has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change does not affect the visibility and contrast of the component against its background.
```diff
-    fill={COLORS.white2}
+    fill={COLORS.white}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->